### DISCUSSION TITLE
Add numbers in the file automatically using python

### DISCUSSION THREAD

**Discussion by deeptamresearchfoundation** [samuelmarina/is-even#132] [Ideas]
We can add numbers in the file using python

**Comment by deeptamresearchfoundation** [comment]
By just make a main.py and open the index.js file
Add the code to main.py : -
also install pyautogui and inflect using pip install in the terminal

the link of the file : https://github.com/deeptamresearchfoundation/main.py

and run this file place the cursor in index.js where we put the numbers
and whoa the number is now adding in the file automatically by the bot ha! ha! ha!

Tip: you can low the value of sleeping time in python to save time but your computer could crash so not low the value of sleeping time

**Reply by Shashank3736** [reply]
Automating things is not a new thing in coding. But in this project, all the lines are written by humans so.... 😞

**Comment by Hocuri** [comment]
I think that's a bad idea. I'm very sure that all the 375000 numbers that are in here were added by hand. Let's not change our process now. Also, I think it's better if a human is in full control over the process, as a computer might incorrectly classify a number sometimes.

**Reply by get-softwares** [reply]
But we can get over about 1 trillion using this method if you don't want it's okay

**Reply by SarangT123** [reply]
> But we can get over about 1 trillion using this method if you don't want it's okay

Well gitlarge cant support that big files

**Reply by PratyushManas** [reply]
I tried to create a simple text file with the code till 696969. The file size shot up to 2.20GB.

**Reply by Fabizocker456** [reply]
Just use this module to check the number's evenness. You still need is-odd for all the odd numbers tho

**Comment by NeuronButter** [comment]
This is not good programming practice. As already identified previously, this line of code is not suitable for production environments:
```py
    if(x % 2 == 0):
        ret = "true"
    else:
        ret = "false"
```
(lines 10-13), as the % operator has not been tested enough.

Current method of adding numbers still remains as best practice for *production* environments. You may use the % operator in your own projects, if you are willing to take the risk.

**Reply by PratyushManas** [reply]
What Samuel has tried here, is to free your minds from the stereotypical methods of programming.
